CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ A new `ChatChannelVC` is introduced that represents the old `ChatMessageListVC`,
 ### ✅ Added
 - For non-DM channels, the avatar is now shown as a combination of the avatars of the last active members of the channel [#1344](https://github.com/GetStream/stream-chat-swift/pull/1344)
 - New DateFormatter methods `rfc3339Date` and `rfc3339DateString` [#1403](https://github.com/GetStream/stream-chat-swift/pull/1403)
+- Add a new `isMentionsEnabled` flag to make it easier to disable the user mentions in the ComposerVC [#1416](https://github.com/GetStream/stream-chat-swift/pull/1416)
```

Comment:
We should name the flag `shouldShowMentions` for SDK parity

Reply:
We already have `ComposerVC.isCommandsEnabled` and `ComposerVC.isAttachmentsEnabled` so it makes sense for this one to have the same pattern. Also, for feature flags it is way more common to use the `is...Enabled` pattern than `should...`.